src/App.tsx

```diff
@@ -1,73 +1,105 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { DropdownMenu } from './components/DropdownMenu';
+import { Person } from './types/Person';
+import cn from 'classnames';
+import '@fortawesome/fontawesome-free/css/all.css';
+import debounce from 'lodash.debounce';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [isDropdownActive, setIsDropdownActive] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  const applyQuery = useCallback(debounce(setAppliedQuery, 300), []);
```

Comment:
![image](https://github.com/mate-academy/react_autocomplete/assets/56752562/53865cf0-2d3b-4b48-aaec-ef24a91d3ed7)
Try to fix all linter errors